tests/Functional/Schema/ComparatorTest.php

```diff
@@ -22,7 +22,7 @@ protected function setUp(): void
         parent::setUp();
 
         $this->schemaManager = $this->connection->getSchemaManager();
-        $this->comparator    = new Comparator();
+        $this->comparator    = new Comparator($this->schemaManager->getDatabasePlatform());
```

Comment:
Please do not modify existing tests. All existing tests should pass without changes.